pkg/server/singleprocess/service_job.go

```diff
@@ -737,12 +737,14 @@ func (s *Service) GetJobStream(
 
 			// Updated job, requery it
 			ws = memdb.NewWatchSet()
-			job, err = s.state(ctx).JobById(ctx, job.Id, ws)
+
+			updatedJobjob, err := s.state(ctx).JobById(ctx, job.Id, ws)
```

Comment:
I kinda love the name `updatedJobjob` but could it just be `updatedJob`?

Reply:
🤦

Reply:
I almost suggested camel-casing it, thinking that @izaaklauer intended to call it "jobjob" 😆